src/helpers.coffee

```diff
@@ -3,6 +3,21 @@
 # arrays, count characters, that sort of thing.
 
 
+# pollyfill for async/await
```

Comment:
There is, because the `async.coffee` tests return promises and require `await` or some alternative like the helper function when running in Node 7+. Even if those tests are skipped at times, there still needs to be code in the file that supports running those tests. Since we can't use `await` anywhere in the file, and I would rather not use callbacks, `yield` + the helper is the best alternative.

The helper function is 13 lines of code, is this really worth arguing over?